robocopkaka/event-manager-rails
# #163599240 fix yarn issues on heroku

#### What does this PR do?
Fix issues when deploying to Heroku
#### Description of Task to be completed?
* Fix issues with Yarn on Heroku
* Fix issue where Rails isn't rendering `public/index.html` because of changes to how `render` works in `ActionController::API`
* Move `devDependencies` to `dependencies` object
#### How should this be manually tested?
* You should be able to change URLs' when visiting `https://shielded-citadel-82961.herokuapp.com/` and have the appropriate component load up
#### Any background context you want to provide?
* Also fixed issues around the app not working when you tried to navigate to a different page using a URL
#### What are the relevant pivotal tracker stories? (if applicable)
[#163599240](https://www.pivotaltracker.com/story/show/163599240)
